packages/autotest/src/autotest/ClassPortal.ts

```diff
@@ -282,23 +276,6 @@ export class ClassPortal implements IClassPortal {
                 // TODO: this could actually be sent to the frontend for consideration in the course-specific classy controller
                 const gradeRecord = res.output.report;
                 feedback = gradeRecord.feedback;
```

Comment:
No more custom feedback from the looks of it. Always need it on the Result Record `feedback` attribute.

Reply:
We don't do feedback mode anymore because it is simpler to just let the container have full control over the feedback; letting Classy make decisions was problematic (and only 310 used it anyways).

Reply:
Just to be clear for future me: We don't do `feedbackMode` anymore, but the grade and results objects still have `custom` fields that are persisted and can be used to store course-specific data.

Reply:
Yeah, this looks great. Way simpler.